samza-core/src/test/java/org/apache/samza/example/PageViewCounterExample.java

```diff
@@ -42,32 +40,29 @@
 public class PageViewCounterExample implements StreamApplication {
 
   @Override public void init(StreamGraph graph, Config config) {
+    MessageStream<PageViewEvent> pageViewEvents =
+        graph.getInputStream("pageViewEventStream", (k, m) -> (PageViewEvent) m);
+    OutputStream<String, PageViewCount, PageViewCount> pageViewEventPerMemberStream = graph
+        .getOutputStream("pageViewEventPerMemberStream", m -> m.memberId, m -> m);
```

Comment:
Thanks. No strong preference on this one!  I suspect the inlined version is slightly easier for type inference. But, that's a very minor/trivial convenience.